community/kernel/src/main/java/org/neo4j/kernel/internal/GraphDatabaseAPI.java

```diff
@@ -48,6 +52,43 @@
      */
     URL validateURLAccess( URL url ) throws URLAccessValidationError;
 
-    @Deprecated
```

Comment:
I have removed the deprecated because the method is used and now `GraphDatabaseAPI` is an internal class. Sure we can do the change you are proposing since it is internal API, but I would prefer to take it on another PR.